contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -26,7 +27,9 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint96 juelsPerGas,
     uint256 l1FeeShareWei,
     uint96 callbackCostJuels,
-    uint96 totalCostJuels
+    uint72 donFee,
```

Comment:
nit: needs denomination (USDCents, Wei, ...)

Reply:
added "juels" to these 3